sass/blocks/_blocks.scss

```diff
@@ -84,6 +84,23 @@
 	}
 }
 
+//! Overwrite inline styles of classic editor iframe embeds
+.entry-content > iframe,
+.entry-summary > iframe {
+
+	margin: 32px $size__spacing-unit !important;
```

Comment:
In general this patch looks good, but is there any way we can accomplish this without the use of `!important`? There may not be in this case, but I always just make sure to ask. 🙂

Reply:
You might just need to make the selector more specific.

Reply:
@kjellr and @joyously Believe me, I don't like `!important` statements either. Unfortunaterly, using the `!important` statement is the only way to fix this issue as I need to overwrite CSS that had been added directly to the iframe element:

> <iframe [...] style="background: white; max-width: 640px; width: calc(100% - 2px); border-radius: 3px; border: 1px solid rgb(219, 219, 219); box-shadow: none; display: block; margin: 0px 0px 12px; min-width: 326px; padding: 0px;"></iframe>

Reply:
Seems like it would be a padding on `entry-content` instead of a margin on everything else.

Reply:
> @kjellr and @joyously Believe me, I don't like !important statements either. Unfortunaterly, using the !important statement is the only way to fix this particular issue, inline styles of the iframe element need to be overwritten

I did some digging as well, and came to the same conclusion. A last (but potentially weird?) resort would be to change the selector from `iframe` to `iframe[style]`, which would only target frames that have inline styles added to them. This would narrow the scope a little bit. Mind giving that a try?

> Seems like it would be a padding on entry-content instead of a margin on everything else.

The reason we don't currently apply margins to `entry-content` is so that we can more elegantly handle Gutenberg's wide & full alignment blocks. This is also used in the [Gutenberg Starter Theme](https://github.com/WordPress/gutenberg-starter-theme), and is a recommended method as per @jasmussen's [demo](https://codepen.io/joen/pen/zLWvrW) from the [Gutenberg docs](https://wordpress.org/gutenberg/handbook/extensibility/theme-support/#wide-alignments-and-floats). We could definitely work out a different solution (this isn't the way @allancole and I handled Gutenberg styles in [the Music theme](https://themeshaper.com/2018/05/22/music-a-gutenberg-powered-theme/) for instance), but it's been mostly working well except for this (relatively small) issue at this point.

Reply:
> Seems like it would be a padding on `entry-content` instead of a margin on everything else.

@joyously I’m not sure I understand what you meant. Would you mind explaining it to me? 

> I did some digging as well, and came to the same conclusion. A last (but potentially weird?) resort would be to change the selector from `iframe` to `iframe[style]`, which would only target frames that have inline styles added to them. This would narrow the scope a little bit. Mind giving that a try?

@kjellr Sure, that sounds like an elegant and save approach. 😉

Reply:
> I’m not sure I understand what you meant. Would you mind explaining it to me?

In order for the content to have white space on both sides, margins are being applied to each kind of content. An alternative would be to apply padding to `entry-content` instead, and use negative margins for the things that are wide. 
But margins on all is the direction we are going.